packages/components/src/components/base/models/constants.ts

```diff
@@ -22,22 +22,22 @@ export enum GRID_CHECKBOX_OPTIONS {
     NONE,
 }
 
-export const PermissionTypes = {
+export enum PermissionTypes {
```

Comment:
Does it make sense to move this to labkey-api-js?

Reply:
> Switch PermissionTypes to an enum. I may still consider moving this to @labkey/api and deprecating Security.effectivePermissions.

I ended up following up with this and moved it to `@labkey/api`.